sodapy/__init__.py

```diff
@@ -229,6 +229,40 @@ def replace(self, dataset_identifier, payload, content_type="json"):
 
         return self._perform_update("put", resource, payload)
 
+    def createNonDataSet(self, payload, files, content_type="json"):
```

Comment:
In this case, the `payload` isn't actually the payload (the file is the payload), so maybe rename `payload` to `params`. `content_type` can be removed, it's irrelevant here.